tiledb/fragment.cc

```diff
@@ -109,6 +109,13 @@ class PyFragmentInfo {
 
   template <typename T>
   py::tuple get_non_empty_domain(py::object schema, uint32_t fid, T did) const {
+    py::bool_ isvar = get_dim_isvar(schema.attr("domain"), did);
```

Comment:
Does this need to be templated? Seems like it is only called for `uint32_t`

Reply:
Yes it should be able to take in both the dimension's index the dimension's name.

Reply:
I see, thanks 👍